src/components/Icon/index.js

```diff
@@ -34,21 +39,26 @@ const typeToSvgIcon = {
   'cancel': FaTimesCircle,
   'check': FaCheck,
   'chevron-down': FaChevronDown,
-  'chevron-right': FaChevronRight,
   'chevron-left': FaChevronLeft,
+  'chevron-right': FaChevronRight,
   'clock': FaClockO,
   'close': FaClose,
   'course': FaFolderOpenO,
   'dollar': FaMoney,
+  'home': FaHome,
   'info': FaInfoCircle,
   'lesson': FaFileO,
+  'list-ul': FaListUl,
   'menu': FaBars,
   'play': FaPlay,
   'question': FaQuestionCircle,
   'refresh': FaRefresh,
   'remove': FaMinusCircle,
   'search': FaSearch,
+  'sign-out': FaSignOut,
+  'slack': FaSlack,
   'success': FaCheckCircle,
+  'user': FaUser,
   'warning': FaExclamationCircle,
 }
```

Comment:
We could potentially programmatically make these keys in the future if this becomes difficult to maintain the mappings